api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -771,6 +771,39 @@ def execute_profile(
             block_max_volume=block_max_volume,
         )
 
+    @publish(command=cmds.thermocycler_start_execute_profile)
+    @requires_version(2, 27)
+    def start_execute_profile(
+        self,
+        steps: List[ThermocyclerStep],
+        repetitions: int,
+        block_max_volume: Optional[float] = None,
+    ) -> Task:
+        """Execute a Thermocycler profile and returns a :py:class:`Task` representing its execution.
```

Comment:
Nitpicky phrasing suggestion:

```suggestion
        """Start a Thermocycler profile and return a :py:class:`Task` representing its execution.
```